python/lsst/ts/fiberspectrograph/config_schema.py

```diff
@@ -38,8 +38,12 @@
     type: string
     default: "summit"
     pattern: "^[a-z0-9][.a-z0-9]*[a-z0-9]$"
+  image_service_url:
```

Comment:
Same comment as the one I made in the electrometer PR. You may want to increase the config schema version.